llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -34,6 +34,63 @@ static cl::opt<unsigned> SLPMaxVF(
         "exclusively by SLP vectorizer."),
     cl::Hidden);
 
+InstructionCost
+RISCVTTIImpl::getRISCVInstructionCost(ArrayRef<unsigned> OpCodes, MVT VT,
+                                      TTI::TargetCostKind CostKind) {
+  size_t NumInstr = OpCodes.size();
+  if (CostKind == TTI::TCK_CodeSize)
```

Comment:
I assume we've now changed the behavior of CostKind==TTI::TCK_CodeSize by removing all the getLMULCost calls? Can we test that?

Reply:
I update the related shuffle tests.

Reply:
Then it is no longer a NFC. :-)

Reply:
Yeah, I just forgot this.. I'll rename the title.